mmdeploy/codebase/mmdet3d/deploy/voxel_detection_model.py

```diff
@@ -84,22 +81,15 @@ def forward(self,
         Returns:
             list: A list contains predictions.
         """
-        result_list = []
-        for i in range(len(img_metas)):
-            voxels, num_points, coors = VoxelDetectionModel.voxelize(
-                self.model_cfg, points[i])
-            input_dict = {
-                'voxels': voxels,
-                'num_points': num_points,
-                'coors': coors
-            }
-            outputs = self.wrapper(input_dict)
-            result = VoxelDetectionModel.post_process(self.model_cfg,
-                                                      self.deploy_cfg, outputs,
-                                                      img_metas[i],
-                                                      self.device)[0]
-            result_list.append(result)
-        return result_list
+        preprocessed = inputs['voxels']
+        input_dict = {
+            'voxels': preprocessed['voxels'].to(self.device),
+            'num_points': preprocessed['num_points'].to(self.device),
+            'coors': preprocessed['coors'].to(self.device)
+        }
+        outputs = self.wrapper(input_dict)
+
+        return [outputs]
```

Comment:
could we call the `VoxelDetectionModel.postprocess` and return the result in `forward`. Then the output of backend would be aligned with that from PyTorch model.

Reply:
done.